astropy/utils/console.py

```diff
@@ -500,20 +532,10 @@ def __init__(self, total_or_items, ipython_widget=False, file=None):
             to detect the IPython console), the progress bar will be
             completely silent.
         """
-
-        if ipython_widget:
```

Comment:
Note: I removed this because the imported stuff is not used here; the import is tried anyway at the end, in the `self.update` call.